### ISSUE TITLE
Can not create VMI because Image size is not a multiple of request alignment

### ISSUE THREAD

**Issue by Huimintai** [kubevirt/kubevirt#8473]
**What happened**:
When I create a VM attach  a 2T data volume,I can not create the VMI successfully with this error:
<img width="1301" alt="image" src="https://user-images.githubusercontent.com/11571376/190356509-716e30e1-6026-4447-b4d5-ed223b7253b3.png">

But when create a smaller such as 50G data volume it can create successfully.
**Additional context**:

**Environment**:
- KubeVirt version (use `virtctl version`): 
```
# virtctl version
Client Version: version.Info{GitVersion:"v0.50.0", GitCommit:"7e034450c10ad2a879db960b9912be2dff7ed9ce", GitTreeState:"clean", BuildDate:"2022-02-10T13:39:37Z", GoVersion:"go1.17.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{GitVersion:"v0.50.0", GitCommit:"7e034450c10ad2a879db960b9912be2dff7ed9ce", GitTreeState:"clean", BuildDate:"2022-02-10T13:39:37Z", GoVersion:"go1.17.5", Compiler:"gc", Platform:"linux/amd64"}
```
- VM specifications: 
```
# kubectl describe vm largest
Name:         largest
Namespace:    default
Labels:       kubevirt.io/domain=largest
Annotations:  description:
              kubevirt.io/latest-observed-api-version: v1
              kubevirt.io/storage-observed-api-version: v1alpha3
              tkestack.io/image-display-name: Windows Server 2016 64位
API Version:  kubevirt.io/v1
Kind:         VirtualMachine
Metadata:
  Creation Timestamp:  2022-09-07T11:11:36Z
  Generation:          1
  Managed Fields:
    API Version:  kubevirt.io/v1
    Fields Type:  FieldsV1
    fieldsV1:
      f:metadata:
        f:annotations:
          .:
          f:description:
          f:kubevirt.io/latest-observed-api-version:
          f:kubevirt.io/storage-observed-api-version:
          f:tkestack.io/image-display-name:
        f:labels:
          .:
          f:kubevirt.io/domain:
      f:spec:
        .:
        f:dataVolumeTemplates:
        f:running:
        f:template:
          .:
          f:metadata:
            .:
            f:annotations:
            f:labels:
          f:spec:
            .:
            f:domain:
              .:
              f:cpu:
                .:
                f:cores:
              f:devices:
                .:
                f:disks:
                f:inputs:
                f:interfaces:
              f:machine:
                .:
                f:type:
              f:resources:
                .:
                f:requests:
                  .:
                  f:memory:
            f:networks:
            f:volumes:
    Manager:      Mozilla
    Operation:    Update
    Time:         2022-09-07T11:11:36Z
    API Version:  kubevirt.io/v1alpha3
    Fields Type:  FieldsV1
    fieldsV1:
      f:status:
        .:
        f:conditions:
        f:printableStatus:
        f:startFailure:
          .:
          f:consecutiveFailCount:
          f:lastFailedVMIUID:
          f:retryAfterTimestamp:
        f:volumeSnapshotStatuses:
    Manager:         Go-http-client
    Operation:       Update
    Time:            2022-09-15T08:44:50Z
  Resource Version:  17998501
  UID:               5c9027bf-7c93-4b91-aad0-53731850cb90
Spec:
  Data Volume Templates:
    Metadata:
      Creation Timestamp:  <nil>
      Name:                rootfs.largest
    Spec:
      Pvc:
        Access Modes:
          ReadWriteMany
        Resources:
          Requests:
            Storage:         50Gi
        Storage Class Name:  csi-cephfs-sc
        Volume Mode:         Filesystem
      Source:
        Pvc:
          Name:       win16huatuo
          Namespace:  kube-public
    Metadata:
      Creation Timestamp:  <nil>
      Name:                datavolume1.largest
    Spec:
      Pvc:
        Access Modes:
          ReadWriteMany
        Resources:
          Requests:
            Storage:         2Ti
        Storage Class Name:  csi-cephfs-sc
        Volume Mode:         Filesystem
      Source:
        Blank:
  Running:  true
  Template:
    Metadata:
      Annotations:
        tkestack.io/image-display-name:  Windows Server 2016 64位
      Creation Timestamp:                <nil>
      Labels:
        kubevirt.io/domain:  largest
    Spec:
      Domain:
        Cpu:
          Cores:  4
        Devices:
          Disks:
            Boot Order:  1
            Disk:
              Bus:       virtio
            Name:        rootfs
            Boot Order:  2
            Disk:
              Bus:  virtio
            Name:   datavolume1
          Inputs:
            Bus:   usb
            Name:  tablet
            Type:  tablet
          Interfaces:
            Bridge:
            Model:  virtio
            Name:   default
        Machine:
          Type:  q35
        Resources:
          Requests:
            Memory:  8G
      Networks:
        Name:  default
        Pod:
      Volumes:
        Data Volume:
          Name:  rootfs.largest
        Name:    rootfs
        Data Volume:
          Name:  datavolume1.largest
        Name:    datavolume1
Status:
  Conditions:
    Last Probe Time:       2022-09-15T08:44:50Z
    Last Transition Time:  2022-09-15T08:44:50Z
    Message:               VMI does not exist
    Reason:                VMINotExists
    Status:                False
    Type:                  Ready
    Last Probe Time:       <nil>
    Last Transition Time:  <nil>
    Message:               cannot migrate VMI which does not use masquerade to connect to the pod network
    Reason:                InterfaceNotLiveMigratable
    Status:                False
    Type:                  LiveMigratable
  Printable Status:        CrashLoopBackOff
  Start Failure:
    Consecutive Fail Count:  367
    Last Failed VMIUID:      b49382e2-6580-4a77-9154-599eb546ee59
    Retry After Timestamp:   2022-09-15T08:49:50Z
  Volume Snapshot Statuses:
    Enabled:  false
    Name:     rootfs
    Reason:   No VolumeSnapshotClass: Volume snapshots are not configured for this StorageClass [csi-cephfs-sc] [rootfs]
    Enabled:  false
    Name:     datavolume1
    Reason:   No VolumeSnapshotClass: Volume snapshots are not configured for this StorageClass [csi-cephfs-sc] [datavolume1]
Events:
  Type    Reason            Age                       From                       Message
  ----    ------            ----                      ----                       -------
  Normal  SuccessfulDelete  51m                       virtualmachine-controller  Stopped the virtual machine by deleting the virtual machine instance a0e1c7b2-b4b6-46a8-824a-48050eed2d03
  Normal  SuccessfulDelete  39m                       virtualmachine-controller  Stopped the virtual machine by deleting the virtual machine instance 08c72296-26d0-4b30-b17f-250958406f4f
  Normal  SuccessfulDelete  30m                       virtualmachine-controller  Stopped the virtual machine by deleting the virtual machine instance c508b63c-539c-4c0b-9542-c879dd82fe35
  Normal  SuccessfulDelete  18m                       virtualmachine-controller  Stopped the virtual machine by deleting the virtual machine instance 64f8471f-6908-487a-8c21-c2a411002331
  Normal  SuccessfulDelete  10m                       virtualmachine-controller  Stopped the virtual machine by deleting the virtual machine instance bb768c6c-6253-4caf-8f11-e89bc1cfa48a
  Normal  SuccessfulCreate  5m59s (x1194 over 7d21h)  virtualmachine-controller  Started the virtual machine by creating the new virtual machine instance largest
  Normal  SuccessfulDelete  4m29s                     virtualmachine-controller  Stopped the virtual machine by deleting the virtual machine instance b49382e2-6580-4a77-9154-599eb546ee59

```

**Comment by vasiliy-ul** [comment]
I think this has already been fixed. You probably need to update from `v0.50.0` to some recent version.

**Comment by Huimintai** [comment]
> I think this has already been fixed. You probably need to update from `v0.50.0` to some recent version.

Do you mean this fix?I found this PR https://github.com/kubevirt/kubevirt/pull/5806
I use v0.50.0,this fix has already cherrypicked into v0.50.0.

**Comment by vasiliy-ul** [comment]
I mean this https://github.com/kubevirt/kubevirt/pull/6754. I am just not sure if eventually it landed in `v0.50.0`.

**Comment by Huimintai** [comment]
> I mean this #6754. I am just not sure if eventually it landed in `v0.50.0`.

Thanks for your help.I checked in `v0.50.0` didn't include this fix.
